nightly/index.php

```diff
@@ -16,14 +16,19 @@
 
 <p> <strong>Current release series</strong>
 
-<li><a href="v4.1.x/"><strong>v4.1.x series</strong></a>: These are
-images from the stable v4.1.x series.
-<strong><font color="red">This
-is the recommended series for all users to download and
-use.</font></strong></li>
+<li><a href="v5.0.x/"><strong>v5.0.x series</strong></a>: These are
+images from the stable v5.0.x series.
+<strong><font color="red">This is the recommended series for all users
+to download and use.</font></strong></li>
+
 
 <p> <strong>Prior stable release series</strong>
 
+<li><a href="v4.1.x/"><strong>v4.1.x series</strong></a>: These are
+images from the prior stable v4.1.x series.
+<strong><font color="red">This code base is still
+supported.</font></strong></li>
+
 <li><a href="v4.0.x/"><strong>v4.0.x series</strong></a>: These are
 images from the prior stable v4.0.x series.
```

Comment:
Let's move v4.0.x into the "Older" sections (both in the main text content and in the left-hand nav), and just have v4.1.x in the "Prior stable" category.

Reply:
So we're officially not in bugfix on v4.0? I can remove that text?

Reply:
i'd vote for moving v4.0.x out of still supported and into older versions

Reply:
Sorry I deleted my comment asking about the text bug-fix-only mode. It said that v4.0 is still supported but in bug fix only mode. Unless we really want to not support it anymore, I think it should remain here.

Reply:
Alternatively, I can move it to older version, but leave the text

Reply:
We discussed this in Slack: perhaps I was too aggressive here.  Let's leave it in the "prior stable" section for now.  We can always file another PR later if we want to move 4.0 out of this section.